doc/Command-Reference.md

```diff
@@ -4079,4 +4079,35 @@ This command displays the routing policy that takes precedence over the other ro
 		Exit routemap
   ```
 
+**Syslog config commands 
+
+This sub-section of commands is used to add or remove the configured syslog servers.
+
+**config syslog add 
```

Comment:
Surround entire command with `**` to make it bold to match style of rest of document (`**config syslog add**`).

Reply:
fixed.